md/ts-jsdoc.md

```diff
@@ -11,7 +11,15 @@ Add types configuration to your package.json:
 ```json
 "types": "dist/src/index.d.ts",
 "typesVersions": {
-  "*": { "src/*": ["dist/src/*", "dist/src/*/index"] }
+  "*": {
+    "src/*": [
+      "dist/src/*", 
+      "dist/src/*/index"
```

Comment:
maybe this import path issue is caused by this line ??

Reply:
This line works around https://github.com/microsoft/TypeScript/issues/41284, but evidently `src/*` pattern does not match import paths `lib/src/`, hence workaround in this pull.